csrankings-a.csv

```diff
@@ -738,7 +739,7 @@ Alexander S. Kotov,Wayne State University,http://www.cs.wayne.edu/kotov,83gsCFsA
 Alexander Schill,TU Dresden,http://www.rn.inf.tu-dresden.de,TABb2iQAAAAJ
 Alexander Schlaefer,Hamburg University of Technology,https://mtec.et8.tuhh.de/staff/alexander-schlaefer.html,NOSCHOLARPAGE
 Alexander Schliep,Chalmers/GU,https://schlieplab.org,NOSCHOLARPAGE
-Alexander Schwarzmann,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=ASCHWARZMANN,gQo0AHYAAAAJ
+Alexander A. Schwarzmann,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=ASCHWARZMANN,gQo0AHYAAAAJ

```

Comment:
Please add entry for other alias (see https://dblp.org/pid/s/AlexanderAShvartsman.html).